### PR TITLE
ui: fix link between NS, stop simulation afte…

### PR DESCRIPTION
**Pull request by safchain**
…r 2sec

Disable link animation as it consumes lot of CPU.

### CODE REVIEW

**Comment by safchain** [comment]
@sergeyglazyrindev-samsung the goal of this was not to be merged. it is the result of some UI improvement that was needed for a demo. That is going to be used by the network service mesh project. I thought it would be nice to keep it as a WIP so that users can test if the result is better.

**Comment by safchain** [comment]
run skydive-python-tests

File: statics/js/components/topology.js [hunk 1]
```diff
@@ -143,6 +143,24 @@ var TopologyComponent = {
...
+          <button id="pin-all" type="button" class="btn btn-primary" \
+                  title="Collapse" @click="pinAll">\
+            <span class="expand-icon-stack">\
+              <i class="glyphicon glyphicon-pushpin icon-main"></i>\
+            </span>\
+            <span class="expand-icon-stack">\
+              <i class="glyphicon glyphicon-ok-circle icon-sub"></i>\
+            </span>\
+          </button>\
+          <button id="pin-all" type="button" class="btn btn-primary" \
```
Comment (matrohon) [inline statics/js/components/topology.js]: It should be unpin-all

**Comment by matrohon** [comment]
I doesn't seem to work as I expected. With #1647, when I highlight g.E().HasKey('RelationType'), every node and edge is highlighted, but with this patch, only nodes are highlighted.

**Pull request merged**